src/index.ts

```diff
@@ -155,7 +161,7 @@ class State implements IState {
     if (!this._simplify) return;
     // We will attempt to change `x_(i)` into `x_i`
     const simple = this._value.slice(this._lastFunction);
-    if (simple.length === 3 || simple.match(/^\([a-zA-Z]*\)$/)) {
+    if (simple.match(/^\([a-zA-Z0-9]*\)$/)) {
```

Comment:
The `simple.length === 3` was breaking things... Just removing it did not cause any test cases to fail, so maybe it's not necessary?